lib/chartist-plugin-legend.js

```diff
@@ -221,12 +226,31 @@
                 legendSeries.forEach(function(seriesIndex) {
                     seriesMetadata[seriesIndex].legend = i;
                 });
+    
+                var active = true;
+                if (options.legendState) {
+                    var found = options.legendState.find(element => element.text == legendText);
+                    if (found) {
+                        active = found.active;
+                    }
+                    else {
+                        // If that location is not in the saved state, only make it
+                        // selected if everything that was saved is selected
+                        active = !options.legendState.find(element => !element.active);
```

Comment:
interesting.

Reply:
This was a tricky case. When tweaking series result in a slightly different list, we need different handling for two cases:

1. If all the known locations were selected, we select the unknown ones
2. If any of the known ones were *not* selected, we don't select the unknown one

At least that's what made sense to me!